### ISSUE TITLE
How to predict trained model to other datasets?

### ISSUE THREAD

**Issue by hdnminh** [MIC-DKFZ/nnUNet#1921]
**Hi Team,**

I have 4 datasets: A, B, C, and D. They are different datasets, but have the same organ: the prostate.

I do not have test data. So I just want to use the model that:
- was trained on dataset **A** to get dice score on the **validation** set of A, B, C, D datasets. 
- was trained on dataset **B** to get dice score on the **validation** set of A, B, C, D datasets. 
- was trained on dataset **C** to get dice score on the **validation** set of A, B, C, D datasets. 
- was trained on dataset **D** to get dice score on the **validation** set of A, B, C, D datasets. 

I do this to verify the domain shift problem between datasets. 

Can you help me how to figure out do that? Thank you so much. I do appreciate your time.

**Comment by hdnminh** [comment]
![image](https://github.com/MIC-DKFZ/nnUNet/assets/76483242/45157e2b-b6dc-42f0-9bd8-4538dd6abbaf)

**Comment by hdnminh** [comment]
Hi @dojoh,

Can you help me? I do appreciate your time and effort. Thank you so much.

**Comment by hdnminh** [comment]
Hi @dojoh 😭

**Comment by dojoh** [comment]
Hi @hdnminh ,
I'm not sure if I get your problem. You have already split your datasets into training and validation?
You could just train them on the individual training sets and run inference on the validation sets?

**Comment by hdnminh** [comment]
Hi @dojoh,

Every dataset was split into 2 parts: training and validation.
To describe more simply what I want:
Step 1: Train a **model X** on the **training set** of dataset A.
Step 2: Evaluate **model X** with the **validation set** of dataset A
Step 3: Moreover, I want to evaluate **model X** above with other "validation sets" of datasets B, C, and D.

**Comment by hdnminh** [comment]
Hi @dojoh

Could you help me, please?

**Comment by dojoh** [comment]
Does https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/how_to_use_nnunet.md#run-inference help you?
